cachito/workers/utils.py

```diff
@@ -19,25 +24,64 @@ def extract_app_src(archive_path, parent_dir):
     return os.path.join(parent_dir, 'app')
 
 
-def get_request_bundle_dir(request_id):
-    """
-    Get the path to the directory where the bundle is being created for the request.
+class RequestBundleDir(PosixPath):
```

Comment:
I don't like the use of the `PosixPath` class since it removes compatibility with Windows. Does the `PurePath` class or something else work?